pkgs/development/python-modules/h5py/default.nix

```diff
@@ -30,18 +30,13 @@ in buildPythonPackage rec {
 
   preBuild = if mpiSupport then "export CC=${mpi}/bin/mpicc" else "";
 
+  checkInputs = optional (isPy26 || isPy27) unittest2;
```

Comment:
`isPy26` test is obsolete, we don't have python26 in nixpkgs anymore.

Reply:
Ok, removed.